hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassCluster.java

```diff
@@ -164,6 +170,8 @@ private void readProperties() {
         keyspace = AlertProperties.getProperty(ALERTS_CASSANDRA_KEYSPACE, "hawkular_alerts");
         cassandraUseSSL = Boolean.parseBoolean(AlertProperties.getProperty(ALERTS_CASSANDRA_USESSL,
                 ALERTS_CASSANDRA_USESSL_ENV, "false"));
+        maxQueue = Integer.parseInt(AlertProperties.getProperty(ALERTS_CASSANDRA_MAX_QUEUE,
+                ALERTS_CASSANDRA_MAX_QUEUE_ENV, "9182"));
```

Comment:
Yeap.
I am not worried, as according of Cassandra driver docs, they are benchmarking 80k request/seconds, so assuming that we can count with a 10% of that and queue-ing efficiently, we can manage the thresholds for single node and scale horizontally.
This PR is not ready finish but my goal is that: to find an acceptable threshold and config that could works as an starting point.